frontend/src/App.css

```diff
@@ -1,38 +1,120 @@
-.App {
-  text-align: center;
+.home {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: space-between;
+}
+
+.sidebar-left {
+  display: flex;
+  background: #000000;
+  width: 56px;
+  height: 100%;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.logo-holder {
+  width: 56px;
+  height: 56px;
+  background: #0061FE;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.sidebar-logo {
+  position: absolute;
+  top: 11px;
+  left: 11px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.content {
   display: flex;
+  flex: 1;
   flex-direction: column;
+}
+
+.content-top {
+  width: 100%;
+  height: 100%;
+  background: #FFFFFF;
+  display: flex;
+  justify-content: center;
   align-items: center;
+  flex-direction: column;
+}
+
+.page-title {
+  color: #1E1919;
+  width: 626px;
```

Comment:
shouldn't we be using percentages instead of hard coded values?

Reply:
eh we can, I just did this to make it cleaner for right now. I think its fine we aren't trying to make this the most robust on all screen sizes and whatnot